src/org/zaproxy/zap/extension/ascanrules/TestCrossSiteScriptV2.java

```diff
@@ -324,6 +324,8 @@ public void scan(HttpMessage msg, String param, String value) {
 								+ " onMouseOver=" + context.getSurroundingQuote() + "alert(1);", context,
 								HtmlContext.IGNORE_TAG);
 						if (contexts2 == null) {
+							log.debug("Failed to find vuln in with simple onmounseover "

```

Comment:
I'm not sure we should log that here, the `contexts2` is (usually) `null` if the scanner is stopped (we don't know if the attack failed or not, we didn't try/check).